sketches/M2_CAN_haz_bus/M2_CAN_haz_bus.ino

```diff
@@ -1,296 +1,661 @@
-// Protocol:
-// PC->Duino:  <size><cmd><data>  where size includes cmd and data but not the size byte itself
-// Duino->PC:  @<size><cmd><data> where size includes cmd and data
-
-/*
-  http://ww1.microchip.com/downloads/en/DeviceDoc/21801e.pdf
-  http://en.wikipedia.org/wiki/CAN_bus
-  http://en.wikipedia.org/wiki/OBD-II_PIDs
-
-  The due_can library files also contain important information.
-  
-  This sketch is configured to work with the 'Macchina' Automotive Interface board 
-  manufactured by RechargeCar Inc. CS_PIN and INT_PIN are specific to this board.
-  
-*/
-// Arduino Due - CAN Sample 1
-// Brief CAN example for Arduino Due
-// Test the transmission from CAN0 Mailbox 0 to CAN1 Mailbox 0
-// By Thibaut Viard/Wilfredo Molina/Collin Kidder 2013
-
-// Required libraries
-#include "variant.h"
 #include <due_can.h>
```

Comment:
in the future, when making changes of this magnitude :) please break code rearrangement (splitting files, rearranging functions in a file, etc) into multiple PR's.  thanks!